spk/python311-wheels/Makefile

```diff
@@ -100,6 +107,15 @@ endif
 
 include ../../mk/spksrc.python.mk
 
+# [atom]
+# Requires
+#  - build:setuptools-scm
+#  - cross:cppy
+ifeq ($(call version_ge, $(TC_GCC), 4.8),1)
+WHEELS += atom==0.10.5
+WHEELS += atom==0.11.0
+endif
```

Comment:
@hgy59 and @mreid-tt just briging your attention that the ability to include wheel requirement definition directly within the WHEELS variable is now functional once again.  This can allow avoiding using multiple requirement files for every gcc versions and ease code maintenance.  Although, I don't suggest moving away from using requirement files, but use this for special cases.

Lastly, note that you cal add the `pure:`, `abi3:` or `cross:` prefix.  With none defaults to `cross`.